### ISSUE TITLE
Rectify the Recyclerview on the Tweet-Harvesting-Fragment

### ISSUE THREAD

**Issue by neelanshsahai** [fossasia/loklak_wok_android#120]
Every time the data is fetched, the Recycler gets scrolled to the bottom most point.

@singhpratyush shall I proceed with this one?

**Comment by singhpratyush** [comment]
@neelanshsahai: I think that a better way to do this would be not scrolling the view and displaying the latest Tweets on the top. What do you think?

**Comment by neelanshsahai** [comment]
Exactly, we can display the latest tweets on the top of recylerview. Its an anomaly that every time data is fetched and the recyclerview gets scrolled to the bottom. Also, I feel the fragment is redundant. I ll check it once and push the changes if the issue gets resolved.
